src/Orleans/AzureUtils/StatsTableDataManager.cs

```diff
@@ -158,7 +170,7 @@ private StatsTableData PopulateStatsTableDataEntry(ICounter statsCounter)
             // order the rows latest-first in the table 
             entry.RowKey = string.Join("$", string.Format("{0:d19}", ticks), name, counterStr);
             entry.DeploymentId = deploymentId;
-            entry.Time = time;
+            entry.Time = now.ToString(DATE_TIME_FORMAT, CultureInfo.InvariantCulture); ;
```

Comment:
Not sure why we'd want to store 'now' as a string, or at all, since the entity already has a timestamp.